problems/arraysWithLoops.js

```diff
@@ -6,7 +6,12 @@
  * @returns {Array} - ["I!", "am!", "a!", "happy!", "array!"]
  */
 
-function shoutForLoop() {}
+function shoutForLoop(Array) {
```

Comment:
your variables should start with lower case letters. Capital letters will be for when we start classes, but great job solving the question!